spec/controllers/sign_out_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe SignOutController, type: :controller do
+  include DfESignInUserHelper
```

Comment:
Can we put some new lines between these to make it a bit easier to read (down to line 19)?